APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+
+* Agreeing upon the types of activities that can be funded,,
+* Establishing a process for deciding which specific activities should be funded,
+* Finding someone to do the work (if necessary),
+* Determining an appropriate compensation level.
+
+The following sections go into detail on each of these points.
+
+What to pay for
+^^^^^^^^^^^^^^^
+We can pay for anything which contributes to the broad mission of the Astropy Project. This includes, for example, code development, community engagement, education, promotion, etc. Funds can be used for many purposes, for example, paid developer time, teaching buyout, development of training materials, expenses for developer meetings or sprints, hosted hardware, and CI or other software services, carbon offsets, promotion and advertising, travel to conferences or meetings to promote Project activities.
+
+How to decide what to pay for
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+1. Pay for things that are important and otherwise won’t get done. *Rationale: there is much more work to be done than there are funds to do all of that work. There are also many volunteers and externally sponsored people interested and willing to do some of the work without compensation from the Project.*
+2. Don’t rely on funding always being there. Plan for sustainability: how will activities be continued when the funds dry up? Invest in documentation, infrastructure, and easy to maintain workflows. For example, funded work on governance should always record the process clearly so that volunteers who follow have less work to do to figure it out. 
+3. Consider potential positive benefits to Astropy maintainers and contributors, maintainers of other projects, end users, and other stakeholders like packagers and educators. 
+4. Also consider the long-term costs to Astropy maintainers and contributors, to counterbalance 3.
+5. Do not increase the load on volunteer maintainers by assuming they are available to review or otherwise support the work. Instead, proposals should include paid time for reviews and other necessary activities. *Rationale: the effect of spending funds should be positive for everyone, not just for the people getting paid. This is also a matter of fairness.*
+
+Restricted / earmarked funding sources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Much of the Project’s funding comes from specific grants from private foundations and government agencies. In these cases, the fundable activities are explicitly described in the proposal text. Such funds must be spent following the proposed plan and often come with further restrictions on who can be funded. For example, NASA funding can only go to individuals or institutions in the USA and EU funding is generally restricted to institutions or individuals in the EU or associated countries. The Finance Committee should make these restrictions clear to the community so that they may submit funding requests aligned with the existing grants.
```

Comment:
```suggestion
Much of the Project’s funding comes from specific grants from private foundations and government agencies. In these cases, the fundable activities are explicitly described in the proposal text. Such funds must be spent following the proposed plan and often come with further restrictions on who can be funded. In particular, government agencies, such as NASA and ESA, may have restrictions on the geographic location of the institutions and individuals receiving funding. The Finance Committee should make these restrictions clear to the community so that they may submit funding requests aligned with the existing grants.
```

Reply:
This is factually untrue. Promises were made to step away from the America first approach, yet here is another example of it.